frontend/components/course-listings/evaluationList.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { useSWRAuth } from '@/components/hooks/useSWRAuth';
+import type { NextPage } from 'next';
+import Head from 'next/head';
+import Image from 'next/image';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+import Card from '@mui/material/Card';
+import Container from '@mui/material/Container';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+import CardContent from '@mui/material/CardContent';
+import CardHeader from '@mui/material/CardHeader';
+import Tabs from '@mui/material/Tabs';
+import Tab from '@mui/material/Tab';
+import Typography from '@mui/material/Typography';
+import Button from '@mui/material/Button';
+import Stack from '@mui/material/Stack';
+import { CourseEvaluationListEntry } from 'utils/api';
+import { Reviews } from '@mui/icons-material';
+import { useRouter } from 'next/router';
+
+type Props = {
+  list: CourseEvaluationListEntry[];
+};
+
+function EvaluationList({ list }: Props) {
+  const router = useRouter();
+
+  const viewDetail = (id: string) => {
+    router.push({
+      pathname: '/course-evaluation',
+      query: { id: id },
+    });
+  };
+
+  return (
+    <Container>
+      <Box
+        sx={{
+          display: 'flex',
+          justifyContent: 'flex-end',
+          pb: 2,
+        }}
+      >
+        <Stack spacing={2} direction="row">
+          <Button variant="contained" sx={{ backgroundColor: '#808080' }}>
+            SHOW COMPLETED
+          </Button>
+          <Button variant="contained" sx={{ backgroundColor: '#00BCD4' }}>
+            CREATE NEW EVALUATION
+          </Button>
```

Comment:
please don't use the CSS sx background color. We created the `CustomTheme.ts` for a reason so we can centralise and make theme consistent.

refer to:
- https://mui.com/material-ui/react-button/#color
- https://mui.com/material-ui/customization/theming/#theme-provider